spec/problem_one_spec.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../problem_one'
+
+describe ProblemOne do
+  let(:p_one) { $p_one = ProblemOne.new }
+
+  it 'should compute for a set of even numbers' do
```

Comment:
Further reading: http://www.betterspecs.org/#should

Reply:
Changed descriptions to match the stated rule: https://github.com/Agilefreaks/Test/pull/42/commits/93c1f7422dca6f49abdf0fbf5ea70e3a92198b85